components/com_content/src/View/Featured/FeedView.php

```diff
@@ -62,11 +62,24 @@ public function display($tpl = null)
 			$link = RouteHelper::getArticleRoute($row->slug, $row->catid, $row->language);
 
 			$description = '';
+
 			$obj = json_decode($row->images);
 
-			if (!empty($obj->image_intro))
+			// Set feed image to image_intro or if that's empty, to image_fulltext
+			$itemImage = '';
+
+			if (!empty($obj->{'image_intro'}))
+			{
+				$itemImage = $obj->{'image_intro'};
+
+			} elseif (!empty($obj->{'image_fulltext'}))
```

Comment:
```suggestion
			}
			elseif (!empty($obj->{'image_fulltext'}))
```
Fix code style errors reported by drone.